posthog/queries/test/__snapshots__/test_trends.ambr

```diff
@@ -307,13 +307,41 @@
               AND (has(['person1'], argMax(person."pmat_name", _timestamp)))) person ON person.id = pdi.person_id
            WHERE team_id = 2
              AND event = 'watched movie'
-             AND timestamp >= toStartOfDay(toDateTime('2019-12-28 00:00:00'), 'UTC')
+             AND timestamp >= toTimezone(toDateTime(toStartOfDay(toDateTime('2019-12-28 00:00:00')), 'UTC'), 'UTC')
              AND timestamp <= toDateTime('2020-01-04 23:59:59') )
         GROUP BY date)
      group by day_start
      order by day_start SETTINGS allow_experimental_window_functions = 1) SETTINGS timeout_before_checking_execution_speed = 60
   '
 ---
+# name: TestFOSSTrends.test_timezone_weekly
+  '
+  
+  SELECT groupArray(day_start) as date,
+         groupArray(count) as data
+  FROM
+    (SELECT SUM(total) AS count,
+            day_start
+     from
+       (SELECT toUInt16(0) AS total,
+               toStartOfWeek(toDateTime('2020-01-26 23:59:59') - toIntervalWeek(number), 0, 'US/Pacific') AS day_start
+        FROM numbers(dateDiff('week', toStartOfWeek(toDateTime('2020-01-12 08:00:00'), 0, 'US/Pacific'), toDateTime('2020-01-26 23:59:59'), 'US/Pacific'))
+        UNION ALL SELECT toUInt16(0) AS total,
+                         toStartOfWeek(toDateTime('2020-01-12 08:00:00'), 0, 'US/Pacific')
+        UNION ALL SELECT count(*) as data,
+                         toStartOfWeek(toDateTime(timestamp), 0, 'US/Pacific') as date
+        FROM
+          (SELECT e.timestamp as timestamp
+           FROM events e
+           WHERE team_id = 2
+             AND event = 'sign up'
+             AND timestamp >= toTimezone(toDateTime(toStartOfWeek(toDateTime('2020-01-12 08:00:00'), 0), 'US/Pacific'), 'UTC')
+             AND timestamp <= toDateTime('2020-01-26 23:59:59') )
```

Comment:
Q: Why don't we need a similar treatment for the upper bound?

Reply:
We don't do anything special to the upper bound. On the lower bound, we do the truncation which is what clears the time figures

Reply:
I guess if I'm in US, and my colleague is in UK, we'll both see different values for the last bucket?

Ah, this doesn't happen because we take the same 'end of day' timestamp for both! But, if it's 11 PM your time, 4 am my time, then basically I'll just have one more bucket for the last day. Eh, guess that's fine. 👍 .